openapi.yaml

```diff
@@ -0,0 +1,698 @@
+swagger: '2.0'
+info:
+  title: 'Game Curator API'
+  description: 'Learning project API to retrieve and manipulate game review data.'
+  version: 'v1'
+  license:
+    name: GNU Affero General Public License Version 3
+    url: http://www.gnu.org/licenses/agpl-3.0.en.html
+  contact:
+    name: IS Data Architecture Team
+    url: https://is.oregonstate.edu/data-architecture
+    email: isdataarchitecture@oregonstate.edu
+schemes:
+  - http
+host: 'api.oregonstate.edu'
+basePath: /v1
+externalDocs:
+  description: GitHub Repository
+  url: https://github.com/osu-mist/game-curator-api
+produces:
+  - application/json
+paths:
+  /games:
+    parameters:
+      - $ref: '#/parameters/authorization'
+    get:
+      summary: List all games
+      description: Get all games
+      tags:
+        - Games
+      parameters:
+        - $ref: '#/parameters/pageNumber'
+        - $ref: '#/parameters/pageSize'
+        - name: developerId
+          in: query
+          type: integer
```

Comment:
Base on [JSON API](https://jsonapi.org/format/#document-resource-object-identification), ID should always be a string.

Reply:
changed all Id instances to strings